### PR TITLE
Player UI layout - first draft

### PR DESCRIPTION
**Pull request by nataphilips**
- Including up-to-date code from the development branch
- Used EpisodesImages component
- The only functional button is play-pause

### CODE REVIEW

File: src/components/common/player/components/player-controls/Button.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { TouchableOpacity } from 'react-native';
+
+import styled from 'styled-components';
+import Icon from '../../../Icon';
```
Comment (nataphilips) [inline src/components/common/player/components/player-controls/Button.js:5]: Do we have a way to reference relative paths? Maybe tilde or root dir?

Comment (steniowagner) [reply src/components/common/player/components/player-controls/Button.js:5]: Not yet, but I'll setup this soon.

File: src/components/common/player/components/player-controls/PlayerControls.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        onPress={() => console.log('Speed up pressed')}
+        name="speedometer"
+        size={35}
+      />
+      <Button
+        onPress={() => console.log('Rewind back pressed')}
+        name="skip-previous-circle"
+        size={55}
+      />
+      {playerState === 'Paused' && (
```
[the remark this answers is not in the document]
Comment (nataphilips) [reply src/components/common/player/components/player-controls/PlayerControls.js]: Done

File: src/components/common/player/components/ProgressBar.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  margin-top: ${({ theme }) => theme.metrics.smallSize}px;
+  font-size: ${({ theme }) => theme.metrics.largeSize}px;
+`;
+
+const ProgressBar = () => {
+  const { position, duration } = useProgress();
+
+  return (
+    <View>
+      <View
```
[the remark this answers is not in the document]
Comment (nataphilips) [reply src/components/common/player/components/ProgressBar.js]: Done

File: src/components/common/player/components/Player.js [hunk 4]
```diff
@@ -21,6 +39,19 @@ const Player = ({ route }) => {
         initialIndex={indexEpisodeSelected}
         playlist={playlist}
       />
+      <EpisodeTitleText>
+        {' '}
```
[the remark this answers is not in the document]
Comment (nataphilips) [reply src/components/common/player/components/Player.js]: Fixed - was a linter problem

**Pull request merged**